dashboard/config/locales/authored_hints.it-IT.yml

```diff
@@ -5113,7 +5113,7 @@ it-IT:
 
           <xml>  <block type="bee_whileNectarAmount">
               <title name="ARG1">nectarRemaining</title>
-              <title name="OP">&gt;</title>
+              <title name="OP">></title>
```

Comment:
Is this change okay?

Reply:
I believe it is

Reply:
I tested it locally and it appears the same to the user.